src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
+     */
+    void kick();
+
+    /**
+     * Kicks the player given a reason.
+     *
+     * @param reason The reason for the kick
+     */
+    void kick(String reason);
```

Comment:
Why `String`? Kick messages can be colored. So it should be `Message` although it has reduced support for everything else.